modules/common/networking/hosts.nix

```diff
@@ -109,12 +109,12 @@ let
 in
 {
   options.ghaf.networking.hosts = {
-    enable = mkOption {
-      type = types.bool;
-      default = true;
-    };
+    enable = lib.mkEnableOption "Ghaf hosts file generation";
```

Comment:
this will now default to false, needs to be checked that it is set in all places

Reply:
Ah, hang on.